### PR TITLE
EVG-2542: improve logging in hostinit processes

### PR DESCRIPTION
**Pull request by tychoish**

### CODE REVIEW

File: hostinit/setup.go [hunk 1]
```diff
@@ -832,6 +848,15 @@ func (init *HostInit) LoadClient(ctx context.Context, target *host.Host) (*LoadC
 		append([]string{"-P", hostSSHInfo.Port}, sshOptions...))
 
 	if err = scpYmlCommand.SetOutput(output); err != nil {
+		grip.Alert(message.WrapError(err, message.Fields{
+			"runner":    RunnerName,
+			"operation": "setting up copy script command",
```
Comment (john-m-liu) [inline hostinit/setup.go]: should this string be different than the one on line 509?

File: subprocess/interface_test.go [hunk 2]
```diff
@@ -66,3 +68,30 @@ func TestOutputOptions(t *testing.T) {
...
+	shouldPass := []OutputOptions{
+		{SuppressError: true, SuppressOutput: true},
+		{Output: buf, SendErrorToOutput: true},
+		{Output: &util.CappedWriter{Buffer: buf, MaxBytes: 1024 * 1024}, SendErrorToOutput: true},
+	}
+
+	assert := assert.New(t) // nolint
+
+	for _, opt := range shouldFail {
+		assert.Error(opt.Validate(), "%+v", opt)
```
Comment (john-m-liu) [inline subprocess/interface_test.go]: This test fails when I run it

--- FAIL: TestOutputOptionsIntegrationTableTest (0.00s)
        Error Trace:    interface_test.go:88
	Error:      	An error is expected but got nil.
	Messages:   	{Output: Error:<nil> SuppressOutput:false SuppressError:false SendOutputToError:false SendErrorToOutput:true}

**Comment by tychoish** [comment]
updated

File: hostinit/setup.go [hunk 3]
```diff
@@ -504,6 +504,14 @@ func (init *HostInit) copyScript(ctx context.Context, target *host.Host, name, s
 		append([]string{"-P", hostInfo.Port}, sshOptions...))
 
 	if err = scpCmd.SetOutput(output); err != nil {
+		grip.Alert(message.WrapError(err, message.Fields{
+			"runner":    RunnerName,
+			"operation": "setting up copy script command",
+			"distro":    target.Distro.Id,
+			"host":      target.Host,
```
Comment (bsamek) [inline hostinit/setup.go]: target.Host.Id? not sure if that's what you meant

Comment (tychoish) [reply hostinit/setup.go]: this is the `host_id` from the database, and we use this field throughout

**Pull request merged**